src/seed/employees.js

```diff
@@ -1,24 +1,34 @@
+import mongoose from 'mongoose';
+
 export default [{
+  _id: mongoose.Types.ObjectId('62891944b389642a7f13ca53'),
   firstName: 'Nicolas',
   lastName: 'Costanza',
   email: 'nicolas22@gmail.com',
   password: 'hola1234',
-  assignedProjects: ['6287b7f40b4757e83aa06ef3', '6223b7f40b4357e81sa06ef3'],
+  assignedProjects: [mongoose.Types.ObjectId('628cf2442fed32f30b57dc75'),
+    mongoose.Types.ObjectId('628cf23dfb13310e147c4fdb')],
```

Comment:
Cambia la indentacion de esta forma:
```suggestion
  assignedProjects: [
    mongoose.Types.ObjectId('628cf2442fed32f30b57dc75'),
    mongoose.Types.ObjectId('628cf23dfb13310e147c4fdb')
  ],
```
Lo mismo para el resto de los assignedProjects